nextflow.config

```diff
@@ -44,56 +44,67 @@ params {
     // General module configuration
     multiqc_config = "$baseDir/configs/multiqc_conf.yaml"
 
-    // Conda
-    enable_conda = false
 }
 
+// Set container registry information
+docker.registry      = 'quay.io'
+singularity.registry = 'quay.io'
```

Comment:
you're missing apptainer, and podman:
https://github.com/nf-core/tools/blob/9ab896c140ee3fde816d250f3d8353aee5547d78/nf_core/pipeline-template/nextflow.config#L183-L189

Reply:
```suggestion
apptainer.registry   = 'quay.io'
docker.registry      = 'quay.io'
podman.registry      = 'quay.io'
singularity.registry = 'quay.io'
```